src/dynamics/video_player/player/player.js

```diff
@@ -508,7 +509,43 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                     "aspect_ratio:aspectratio,fallback-width,fallback-height": function(aspectRatio, fallbackWidth, fallbackHeight) {
                         return aspectRatio || fallbackWidth + "/" + fallbackHeight;
                     },
-                    "containerSizingStyles:aspect_ratio,height,width,floating_height,floating_width,floating_top,floating_right,floating_bottom,floating_left,is_floating": function(aspectRatio, height, width, floatingHeight, floatingWidth, floatingTop, floatingRight, floatingBottom, floatingLeft, isFloating) {
+                    "adsinitialized:adsmanagerloaded,addata,adtagurl,inlinevastxml": function(adsmanagerloaded, addata, adsTagURL, inlineVastXML) {
+                        if (this.get("adsinitialized")) {
+                            if (this.__adInitilizeChecker) this.__adInitilizeChecker.clear();
+                            return true;
+                        }
+                        if (this.get("skipinitial")) {
+                            // on autoplay set to true, we're getting addata from player on ads start or error event
+                            if (typeof addata !== "undefined" && addata) {
```

Comment:
```suggestion
                            if (addata) {
```

Reply:
Hi @sambua, why are we using addata for skipinitial instead of just using adsmanagerloaded on both? The addata comes from the ads manager, so they're related.

Reply:
Hi Natasha, will be better on both case using `addata`, but on `skipinitial` case it's not define, and `adsmanagerloaded` defines before data so, in some case it's possible to see banner if it's only `adsmanagerloaded`